src/BuildEnv/Script.hs

```diff
@@ -404,7 +405,7 @@ stepScript scriptCfg = \case
                 stdoutFile = q ExpandVars ( logPath <.> "stdout" )
                 stderrFile = q ExpandVars ( logPath <.> "stderr" )
             in ( [ "echo \"> " <> unquote cmd <> "\" >> " <> stdoutFile ]
-               , " > " <> stdoutFile <> " 2> >( tee -a " <> stderrFile <> " >&2 )"
+               , " 2>&1 >" <> stdoutFile <> " | tee -a " <> stderrFile
```

Comment:
Doesn't this write both stdout and stderr to `stdoutFile` , and then nothing gets piped to `tee`?

Reply:
No :) Try it!

```
> ghc -e 'putStrLn "stdout" >> System.IO.hPutStrLn System.IO.stderr "stderr"' 2>&1 >/tmp/out | tee -a /tmp/err
stderr
~ > cat /tmp/out
stdout
```

The way shell redirection works is very non-obvious, it took me quite a while to figure this out.

Reply:
Wow, very good.